CHANGES.md

```diff
@@ -1,4 +1,4 @@
-2.0.4 (04-28-2022)
+2.0.4 (23-04-2022)
```

Comment:
I think it will be better if the date is in ISO8601 format.

Reply:
It would be better indeed. We've been keeping historical date formats in the changelog but it's time to change them.